libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostStorage.m

```diff
@@ -74,6 +74,13 @@ - (id)init {
             }];
         }
         
+        //Load from info.plist
+        if([[NSBundle mainBundle] objectForInfoDictionaryKey:@"SFDCOAuthLoginHost"]) {
```

Comment:
I think this should move above the ```MDM``` piece a few lines above, so that ```MDM``` takes precedence? Also, we should probably overwrite the list if ```MDM``` sets it as discussed in the e-mail - not append to the existing list.